### PR TITLE
Enable utility features

### PR DESCRIPTION
**Pull request by EssamEid95000**
this is initial implementation of utility feature based on requirements.

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (essaneid95) [inline util/config.txt]: This config value should be 567.123 was the old value.

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+feature.txt
+feature 1 implementation
+
+feature 2 implementation
+
```
Comment (essaneid95) [inline util/feature.txt:6]: Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgets.

**Review by essaneid95 — COMMENTED** [review]
Please fix config values and add the implementation to feature2

File: util/config.txt (hunk 1 above)
Comment (EssamEid95000) [reply util/config.txt]: @essaneid95 see the latest commit for the implementation

File: util/feature.txt (hunk 2 above)
Comment (EssamEid95000) [reply util/feature.txt:6]: @essaneid95 see the latest commit for the implementation

**Review by essaneid95 — APPROVED** [review]
Look good

**Pull request merged**